linkerd/core/src/test/scala/io/buoyant/linkerd/RouterTest.scala

```diff
@@ -96,4 +102,23 @@ servers:
     val DstBindingFactory.Namer(interpreter) = router.params[DstBindingFactory.Namer]
     assert(interpreter.isInstanceOf[TestInterpreter])
   }
+
+  test("with retries") {
+    val yaml =
+      """|protocol: plain
+         |client:
+         |  retries:
+         |    backoff:
+         |      kind: jittered
+         |      maxMs: 10000
```

Comment:
jackson doesn't enforce any sort of requirements, it will just leave null-values for missing fields. for Ints, this is 0 (which is good for minMs).  for options, it's None.